grid/social-media/profiles/maps/twitter.suma

```diff
@@ -0,0 +1,132 @@
+
+profile = "social-media/profiles@1.0"
+provider = "twitter"
+
+// Required Twitter OAuth 2.0 scopes:
+//  - tweet.read
+//  - users.read
+
+map GetProfiles {
+  // https://developer.twitter.com/en/docs/twitter-api/users/lookup/api-reference/get-users
+  http GET "/2/users" {
+    // FIXME: support pagination by checking if profileIds.length > 100 and providing remaining IDs in nextPage
+    request {
+      query {
+        ids = input.profileIds.join(','),
+        'user.fields' = 'id,username,name,description,profile_image_url,url,public_metrics'
+      }
+      headers {
+        authorization = "Bearer " + parameters.accessToken
+      }
+    }
+
+    response 200 "application/json" {
+      profiles = body.data.map((user) => {
+        const metrics = user.public_metrics || {};
+        return {
```

Comment:
I miss mapping of `name` property (Display name of the profile).

Reply:
Good catch, thanks!